Add first draft of the section “joining”

mostafah opened the pull request:
This is the HTML version of the section discussed at #14.

mostafah committed:
Add first draft of the section “joining”

See discussion at #14.

r12a commented:
for those who want to, this can be viewed as an HTML page at 
http://htmlpreview.github.io/?https://github.com/w3c/alreq/blob/6e7673e0831aa8b7151af8fd4198f93d1cf7cac1/index.html

mostafah committed:
Replace the obsolete <strike> tag with <del>

mostafah committed:
Change image widths to 90% to make them easier to view

mostafah committed:
Change Unicode character name syntaxes

Use the tool provided at http://r12a.github.io/pickers/arabic/ for
consistency.

Also, put the Arabic letters in <span> tags and parenthesis.

mostafah commented:
Applying these three new commits, the new preview is available [here](http://htmlpreview.github.io/?https://github.com/w3c/alreq/blob/fc80e11e98a9defc7fa8e5020e1b2537a682bd6f/index.html).

mostafah committed:
Put letter names in a <span> and apply a minimal stlye for it
